src/utils/key.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This file isn't used anywhere in the code right? It just needed to be accessible to register the client? I'm a little worried we might see this isn't referenced then delete it. Can you make a new folder `src/keys` and have them be named "publicKey" and "privateKey" so the link is obvious?

Reply:
moved and renamed the files